onadata/settings/common.py

```diff
@@ -509,6 +510,31 @@ def configure_logging(logger, **kwargs):
     'application/zip',
 ]
 
+CELERY_BEAT_SCHEDULE = {
+    'Mark-Expired-Exports-Failed': {
+        'task':
+        'onadata.apps.viewer.tasks.mark_expired_pending_exports_as_failed',
+        'schedule': crontab(hour='*', minute='0'),
+        'options': {'ignore_result': True}
+    },
+    'Delete-Expired-Failed-Exports': {
+        'task': 'onadata.apps.viewer.tasks.delete_expired_failed_exports',
+        'schedule': crontab(hour='*', minute='10'),
+        'options': {'ignore_result': True}
+    },
+    'celery.backend_cleanup': {
+        'task': 'celery.backend_cleanup',
+        'schedule': crontab(hour='*', minute='30'),
+        'options': {'ignore_result': True}
+    },
+    'Delete-Inactive-Users': {
+        'task': 'onadata.apps.api.tasks.delete_user_async',
+        'schedule': crontab(
+            0, 0, day_of_month='30'),  # Execute on the 30th of every month.
```

Comment:
@WinnyTroy What about the month of February? Can we consider may be the 1st day of every month?

Reply:
I agree with @DavisRayM on having the configuration on a user's local settings file

Reply:
That said, I like the fact that functionality will be scheduled and executed once a month

Reply:
Alright, sure, I'll remove this configurations from here. 
And I see no problem with having this run on the 1st of every month. Let me change that